include/dev/TPS2HB50BQ1.hpp

```diff
@@ -90,13 +92,12 @@ class TPS2HB50BQ1 {
     IO::GPIO& diagSelect1;
     IO::GPIO& diagSelect2;
     IO::ADC& senseOut;
+    ACS71240 currentSensor;
```

Comment:
the name currentSensor is not clear enough.

```suggestion
ACS71240 acs71240;
```